WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -1271,6 +1265,7 @@ extension EditableOrderViewModel {
     enum ScannerError: Error {
         case nilSKU
         case productNotFound
+        case productNotFoundInLocalStorage
```

Comment:
I am wondering, why is a Scanner Error when the product is not found in local storage? More than an error, I imagine that that's a common scenario and I wouldn't call that an error :)

Reply:
Agree, removed on https://github.com/woocommerce/woocommerce-ios/pull/9768/commits/45b3cb045c98a0837b202f8d6dc79349a9825fd1